### DISCUSSION TITLE
Somehow DaisyUI dont work with my Laravel 10 application

### DISCUSSION THREAD

**Discussion by ayog59** [saadeghi/daisyui#2665] [General]
I followed the Tailwind installation steps and the DaisyUI steps too.
Since my file doesnt use modules.export but export default i import it in the tailwind.config.js and use the import instead of require but even with require no DaisyUi components work for me. When i use the CDN method it wont let me Style Checkboxes.

Code of config:
```
import defaultTheme from 'tailwindcss/defaultTheme';
import daisy from 'daisyui';

/** @type {import('tailwindcss').Config} */
export default {
    content: [
        './vendor/laravel/framework/src/Illuminate/Pagination/resources/views/*.blade.php',
        './storage/framework/views/*.php',
        './resources/views/**/*.blade.php',
        './resources/**/*.js',
        './resources/**/*.vue',
    ],

    theme: {
        extend: {
            fontFamily: {
                sans: ['Figtree', ...defaultTheme.fontFamily.sans],
            },
        },
    },

    plugins: [daisy],
};
```

So when i do apply a DaisyUI component it wont get loaded unless i add the CDN but even with that the Checkboxes wont Style, since i couldnt find a way to fix it, maybe you guys have an idea.

**Comment by saadeghi** [comment]
Do Tailwind CSS class names work?

**Comment by ayog59** [comment]
Tailwind Classes Work but the Forms Plugin dont i guess since It should let
you Style the checkboxes but It doesnt

Pouya Saadeghi ***@***.***> schrieb am Fr., 15. Dez. 2023,
13:20:

**Comment by saadeghi** [comment]
Please share a reproduction repo so I can reproduce the issue.

**Comment by mark1502** [comment]
The problem may be your reference to daisyui in the plugins array.

I'm using Daisyui with Laravel 10 (with Inertia and Vue).  Fwiw, here's a copy of my tailwind.config.js file.  DaisyUI is referenced differently in my config file and everything works as expected.
```
const defaultTheme = require('tailwindcss/defaultTheme');

/** @type {import('tailwindcss').Config} */
module.exports = {
    content: [
        './vendor/laravel/framework/src/Illuminate/Pagination/resources/views/*.blade.php',
        './storage/framework/views/*.php',
        './resources/views/**/*.blade.php',
        './resources/js/**/*.vue',
    ],

    theme: {
        extend: {
            fontFamily: {
                sans: ['Figtree', ...defaultTheme.fontFamily.sans],
            },
        },
    },

    plugins: [
        require('@tailwindcss/forms')({
            // strategy: 'base', // only generate global styles
            strategy: 'class', // only generate classes
          }),
        require("daisyui"),
    ],
};
```
Hope this helps

**Comment by ayog59** [comment]
Pretty dumb but i did forget to Run the vite build on the Server so
localhost had the assets but my Server didnt, thanks for that fast Support
anyways def. Looking Forward using daisyui now.

mark1502 ***@***.***> schrieb am Mi., 20. Dez. 2023, 21:22:

**Reply by mark1502** [reply]
I've done that too!  : ) Good luck
